snap/local/wrappers/fetch-oci

```diff
@@ -8,6 +8,12 @@ if ! which microk8s.kubectl; then
     exit 0
 fi
 
+if ! which timeout; then
+    echo "timeout is not installed."
+    exit 0
+fi
+
+timeout 2m sh <<EOT || echo true
```

Comment:
`|| true` should be sufficient here, correct? Or echo "microk8s docker images downloading in bg" if you want to give a user some information.

Reply:
I think you're right, let me change this before it lands. It would be much better to add a message here.